Documents/Changelog/Changelog.md

```diff
@@ -9,6 +9,7 @@
 =======
 
 ## 2024-11-12 - Build 2411 - November 2024
+* Resolved [#1399](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1399), Hard coded colour setting removed from the `KryptonRibbonTab`.
```

Comment:
You might want to put this under a month next year, unless this is going to be pushed to `canary` as well?

Reply:
v95 is ok....?